vars/k8sLocalDeployment.groovy

```diff
@@ -54,6 +54,7 @@ def call(Map config = [:]) {
                             if (status == "Running") {
```

Comment:
Could we add an additional log line before this condition that maybe just prints out what we get back from minikube status?